src/en/wip-systems.md

```diff
@@ -30,10 +30,10 @@ A Juju Environment System (JES), also sometimes shortened to 'Juju System',
 describes the environment that runs and manages the Juju API servers and the
 underlying database.
 
-This initial environment is also called the 'system environment' 
-(or 'bootstrap environment'), and is what is
-created when the bootstrap command is used.  This System environment is a
-normal Juju environment that just happens to have machines that manage Juju.
+This initial environment is also called the 'system environment' (or
+'bootstrap environment'), and is created when the bootstrap command is used.
+This System environment is a normal Juju environment that just happens to have
+machines that manage Juju.
 
```

Comment:
This whole section is confusing to me, particularly with the proliferation of uses for the terms system, environment and boostrap. Are we stuck with JES? Can't we have a Juju Environment Manager or a Juju Environment Controller instead? It becomes easier to differentiate what these entities are then, and the name is more descriptive